### ISSUE TITLE
Not able to reproduce result ? 

### ISSUE THREAD

**Issue by anupam456** [NVIDIA/waveglow#121]
Hi, 

After very similar training for around 580k steps, I am not getting similar audio quality what your pretrained model is giving. 

Am I missing something ( anything that u didn't mention in your paper ) ?

**Comment by HashiamKadhim** [comment]
Hi, just to clarify, did you use LJSpeech?

**Comment by anupam456** [comment]
Yes and on multi gpu (4) with sigma = .7 and BS = 6 per gpu.

**Comment by HashiamKadhim** [comment]
I used a similar private dataset, and was also not able to reproduce the results. @rafaelvalle I believe this is related to my issue as well: [https://github.com/NVIDIA/waveglow/issues/115](url)

**Comment by rafaelvalle** [comment]
Keep training until audio quality stops improving, even if there's seems to be no change to the loss.

**Comment by rafaelvalle** [comment]
We trained the model for 580k iterations with batch size 24.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
